src/eei.cpp

```diff
@@ -866,7 +866,7 @@ string toHex(evm_uint256be const& value) {
     evm_uint256be balance;
     context->fn_table->get_balance(&balance, context, &msg.destination);
     if (safeLoadUint64(balance) < safeLoadUint64(value))
-      throw new OutOfGasException();
+      throw OutOfGasException();
```

Comment:
If `OutOfGasException{}` is the preferred style (see #184), then it should be changed in all instances in this PR.

Reply:
It's preferred in https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax so I would use it.